app/models/user.rb

```diff
@@ -121,8 +121,7 @@ def create_reset_digest
   # Returns true if the given token matches the digest.
   def authenticated?(attribute, token)
     digest = send("#{attribute}_digest")
-    return false if digest.nil?
```

Comment:
You need to keep the explicit false return if the digest is nil (in case other places are explicitly checking if the return value is false